samples/core/guide/autograph.ipynb

```diff
@@ -415,14 +417,14 @@
       "cell_type": "code",
       "source": [
         "@autograph.convert()\n",
-        "def f(x):\n",
-        "  assert x != 0, 'Do not pass zero!'\n",
-        "  return x * x\n",
+        "def inverse(x):\n",
+        "  assert x != 0.0, 'Do not pass zero!'\n",
+        "  return 1.0/x\n",
```

Comment:
Style nit: for expressions, the Google style guide recommends placing a space around operators, e.g. "1.0 / x"; I think it would be useful to be consistent with it in this notebook.

Reply:
+1 we just don't have an integrated notebook linter (yet?).

Done.